metadata.json

```diff
@@ -7,11 +7,9 @@
   "source": "https://github.com/voxpupuli/puppet-python.git",
   "project_page": "https://github.com/voxpupuli/puppet-python",
   "issues_url": "https://github.com/voxpupuli/puppet-python/issues",
-  "description": "Puppet module for Python",
```

Comment:
Why should the description be removed?

Reply:
because it's an obsolete attribute, was deprecated long time ago

Reply:
`summary` is terser that this `description` we remove.  Maybe we can update the summary with this text or rephrase it to indicate the module capabilities?

The module description on GitHub says *Puppet module for installing and managing Python, pip, virtualenvs and Gunicorn virtual hosts.* as the README.  Maybe we can adjust this and sync the sentences?

Reply:
I can remove the change from this PR if you like. I removed the line, because IDE told me it's wrong and I was touching the file anyway. summary suppose to be terse, imho, documentation says "one liner"

Reply:
rebased/changed summary

Reply:
That's great for me, thanks!